src/plugins/validation.ts

```diff
@@ -146,6 +153,52 @@ export class ValidationPlugin {
     };
   };
 
+  /**
+ *
+ * @returns {function(*): (boolean|string)}
+ */
+ isEmail = (
+ ): ((v: string) => string | true | undefined) => {
+   return (v: string) => {
+     if (v && v!==""){  
+       if (/[a-z0-9]+@[a-z]+\.[a-z]{3}/.test(v) === false) {
+         return "Please use standard domain format, like ‘@mail.mil’"
+       } else if (/[a-z0-9]+@[a-z]+\.[gov|mil]/.test(v) === false) {

```

Comment:
Regex needs help. 
1. Domain name part should allow hyphens
2. Username part should not allow spaces
3. Extension should only allow `.mil` or `.gov` but allows `.mild` or `.govtttt`

Reply:
**Item 1 resolved**
 ![image](https://user-images.githubusercontent.com/84856468/165309471-20aa84d5-52e3-45ac-a800-148b8cc6a1ab.png)

**Item 2 resolved**
![image](https://user-images.githubusercontent.com/84856468/165314228-6878c1d4-10ce-4772-b466-eae3cff8c535.png)

**Item 3 resolved**
![image](https://user-images.githubusercontent.com/84856468/165309780-ee3b8fa7-8949-4a48-92fe-8ce9cba0d48d.png)